src/modules/ImageCreation/selectors/imageCreationSelectors.tsx

```diff
@@ -84,10 +84,10 @@ export const getAlertTitleKey = createSelector(
 	},
 );
 
-export const makeGetInputValue = () => (state: any, type: string) => getEntitesFilters(state)[type];
+export const getInputValue = () => (state: any, type: string) => getEntitiesFilters(state)[type];
```

Comment:
why?

Reply:
I don't get it. Could you please be more specific?